wrims-ide/build.gradle

```diff
@@ -6,11 +6,18 @@ plugins{
     id 'biz.aQute.bnd.builder' version '6.4.0'
 }
 
+configurations {
+    windows_x64
+}
+
 dependencies {
     api(project(':wrims-core'))
     implementation project(':third-party')
 
     compileOnly fileTree(dir: '../eclipse-luna-libs', include: ['*.jar'])
+
+    windows_x64('mil.army.usace.hec:javaHeclib:7-IU-3-win-x86_64@zip')
```

Comment:
Is this not supposed to match the latest version of the javaHeclib we're pulling in the third party module? 

hecdll('mil.army.usace.hec:javaHeclib:7-IU-4-win-x86_64@zip')

Reply:
thanks, I forgot to move it to the versions catalog. Updated

Reply:
Looks good!